atd-vze/src/views/Crashes/Crash.js

```diff
@@ -289,22 +289,22 @@ function Crash(props) {
         </Row>
       ) : (
         <div></div>
-      )}
-      {shouldShowFatalityRecommendations && (
-        <Row>
-          <Col>
-            <Recommendations crashId={props.match.params.id} />
-          </Col>
-        </Row>
-      )}
+      )}     
       <Row>
         <Col>
-          <Notes crashId={props.match.params.id} />
+          <CrashCollapses data={data} props={props} />
         </Col>
       </Row>
-      <Row>
+       {shouldShowFatalityRecommendations && (
+        <Row>
         <Col>
-          <CrashCollapses data={data} props={props} />
+          <Recommendations crashId={props.match.params.id} />
+        </Col>
+      </Row>
+      )}
```

Comment:
can you tab this over to make the indentation match line 298

Reply:
It's the little things. Cleaned it up.